src/css/popup-new.css

```diff
@@ -180,3 +186,112 @@ sign-up-panel::after {
   background-size: 120%;
 }
 
+/* Settings */
+
+.fx-relay-settings-toggles {
+  padding: var(--spacingMd);
+  background-color: var(--colorWhite);
+  border-radius: var(--borderRadiusSm);
+  display: flex;
+  flex-direction: column;
+  gap: var(--spacingMd);
+}
+
+.fx-relay-settings-toggle-wrapper {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.fx-relay-settings-toggle-wrapper-spacer {
+  height: 1px;
+  background-color: var(--colorGrey10);
+}
+
+.fx-relay-settings-toggle {
+  display: block;
+  height: 16px;
+  width: 28px;
+  min-width: 28px;
+  position: relative;
+  overflow: hidden;
+  border: none;
+  border-radius: 1.5em;
+  outline: none;
+  background-color: var(--colorGreen50);
+  background-size: 20px;
+  margin: 0;
+  cursor: pointer;
+}
+
+.fx-relay-settings-toggle:checked:hover {
+  background-color: var(--colorGreen60);
+}
+
+.fx-relay-settings-toggle:checked:focus {
+  box-shadow: var(--colorGreen20);
+}
+
+.fx-relay-settings-toggle:checked:active {
+  background-color: var(--colorGreen70);
+}
+
+.fx-relay-settings-toggle::after {
+  content: "";
+  height: 10px;
+  width: 10px;
+  border-radius: 50%;
+  background-color: rgba(255, 255, 255, 1);
+  position: absolute;
+  top: 0px;
+  bottom: 0;
+  margin: auto;
+  right: 4px;
+  transition: all 0.2s ease;
+}
+
+.data-disabled::after,
+.input-icons-disabled::after {
+  left: 4px;
+  right: 18px;
```

Comment:
These magic numbers seem suspicious? Might warrant a comment or other layout technique.

Reply:
Extremely. This was ported over directly from [relay.css](https://github.com/mozilla/fx-private-relay-add-on/blob/main/src/css/relay.css#L225-L230)